apps/qubit/modules/informationobject/templates/inventorySuccess.php

```diff
@@ -49,18 +49,7 @@
               <?php endif; ?>
             </td>
             <td>
-              <?php if (isset($doc['dates'])): ?>
-                <?php foreach ($doc['dates']->getRawValue() as $date): ?>
-                  <?php if (isset($date['startDateString'])
-                    || isset($date['endDateString'])
-                    || null != get_search_i18n($date, 'date', array('culture' => $culture))): ?>
```

Comment:
Oh, it's a date not a title, that's the reason you thought it'd be good to use `allowEmpty`, right?